2017/essence-of-calculus/hebrew/sentence_translations.json

```
@@ -19,91 +19,91 @@
  },
  {
   "input": "The goal here, as the name suggests, is to really get the heart of the subject out in one binge-watchable set.",
-  "translatedText": "המטרה כאן, כפי שהשם מרמז, היא באמת להוציא את לב הנושא בסט אחד שניתן לצפייה בבולמוס.",
+  "translatedText": "המטרה שלי, כפי שהשם מרמז, היא ליצור סדרה אחת קצרה שמכסה את החומר באופן מקיף, ובו בזמן מתעמקת ומגיעה ללב הנושא. ",
   "model": "google_nmt",
   "from_community_srt": "המטרה של הסרטונים האלו היא בעצם לחקור את החדו\"א על בוריה ובכמה סרטונים בלבד.",
-  "n_reviews": 0,
+  "n_reviews": 1,
   "start": 24.3,
   "end": 29.72
  },
  {
   "input": "But with a topic that's as broad as calculus, there's a lot of things that can mean, so here's what I have in mind specifically.",
-  "translatedText": "אבל עם נושא רחב כמו חישוב, יש הרבה דברים שיכולים להיות, אז הנה מה שיש לי בראש ספציפית.",
+  "translatedText": "אבל עם נושא רחב כמו אינפי, למה אנחנו אפילו מתכוונים בלב הנושא? זוהי התוכנית שלי:",
```

אולי

```suggestion
  "translatedText": "אבל עם נושא נרחב כמו אינפי, למה אנחנו אפילו מתכוונים בלב הנושא? זוהי התוכנית שלי:",
```

?

לא בטוח מה נשמע יותר טוב